app/models/lockstep/status.rb

```diff
@@ -0,0 +1,16 @@
+class Lockstep::Status < Lockstep::ApiRecord
```

Comment:
can we add this model to documentation in readme.md

Reply:
Yup will do